packages/vite/LICENSE.md

```diff
@@ -2197,7 +2197,7 @@ Repository: unjs/mlly
 
 > MIT License
 > 
-> Copyright (c) 2022 UnJS
+> Copyright (c) 2021
```

Comment:
I've no idea why this is happening… All I do was rebuilding the project with the updated dependencies.